### PR TITLE
Index and show for book

### PR DESCRIPTION
**Pull request by NicolasLeutwyler**
# Summary

* Routes Index and Show for Book implemented

# Trello Card

* [Card.](https://trello.com/c/Kgc3QxyK/24-crear-rutas-index-y-show-para-book)

### CODE REVIEW

File: wbooks-api/config/routes.rb [hunk 1]
```diff
@@ -4,6 +4,9 @@
 
 	get '/', to: 'landing#index'
 
+  get 'api/v1/books', to: 'books#index'
```
Comment (mdesanti) [inline wbooks-api/config/routes.rb]: This route should be inside https://github.com/wolox-training/nl-rails/pull/26/files#diff-8de07e19439d39f3ea9b09d506646a00R11.
Moreover, you can replace these two lines with: `resources :books, only: [:show, :create]`

File: wbooks-api/spec/factories.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# This will guess the User class
+FactoryGirl.define do
+  factory :user do
+    first_name 'Test'
```
Comment (mdesanti) [inline wbooks-api/spec/factories.rb]: Use `Faker` to fake name

File: wbooks-api/spec/factories.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    last_name  'TestLastName'
```
Comment (mdesanti) [inline wbooks-api/spec/factories.rb]: Use `Faker`

File: wbooks-api/spec/factories.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    email 'test@wolox.com.ar'
```
Comment (mdesanti) [inline wbooks-api/spec/factories.rb]: Use `Faker`

File: wbooks-api/spec/factories.rb [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    password '123123123'
```
Comment (mdesanti) [inline wbooks-api/spec/factories.rb]: Use `Faker`

File: wbooks-api/spec/factories.rb [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+  end
+
+  factory :book do
+    genre 'Fantasy'
```
Comment (mdesanti) [inline wbooks-api/spec/factories.rb]: Use `Faker`

**Comment by NicolasLeutwyler** [comment]
I think it's done

**Pull request merged**